website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -18,9 +18,10 @@ This section describes what API tokens are. For information on how to create the
 
 :::
 
-Use API tokens to connect to the Unleash server API. API tokens come in three distinct types:
+Use API tokens to connect to the Unleash server API. API tokens come in fours distinct types:
 
 -   [Admin tokens](#admin-tokens)
+-   [Personal access tokens](#personal-access-tokens)
 -   [Client tokens](#client-tokens)
 -   [Front-end tokens](#front-end-tokens)
```

Comment:
I think this list should be sorted alphabetically, but am happy to be challenged on that.


```suggestion
-   [Admin tokens](#admin-tokens)
-   [Client tokens](#client-tokens)
-   [Front-end tokens](#front-end-tokens)
-   [Personal access tokens](#personal-access-tokens)
```

Reply:
So the rationale here was that pats and admin tokens are so similar I wanted them grouped - happy to sort this alphabetically if you think that's clearer?

Reply:
That was sorta what I figured. I think the list is short enough that either can work, so I'm not super opposed to leaving it as it was. Another option is putting the list in the same order the sections appear in the text. That might be the most sensible?

Reply:
I'm happy if you're happy. It's already in that order so I'm gonna close this comment thread if you're okay